doc/specs/#194 - PWA Support.md

```diff
@@ -0,0 +1,122 @@
+---
+author: Amrutha Srinivasan @amrutha95
+created on: 2020-08-19
+last updated: 2020-08-19
+issue id: 164
+---
+
+# Spec Title
+
+For [#164](https://github.com/microsoft/winget-cli/issues/164)
+
+## Abstract
+
+This feature adds PWA (Progressive Web Application) support to winget. The goal is to allow users to install and maintain PWAs just like they would any other type of application on winget. As a first step, this would be available as an "experimental" feature to make it available to the community and receive feedback.
+
+## Inspiration
+
+PWA adoption is extremely important in the Windows app ecosystem. It is also crucial that they are treated as first class citizens in the app ecosystem, just like any native application. Providing PWA support for winget is key to achieving this goal.
+
+## Solution Design
+
+### System requirements
+
+Windows 10 v2004 or newer  

```

Comment:
The client is intended to work back to 1709, how will this be validated in the client to ensure users on earlier unsupported versions of Windows 10?

Reply:
If the YAML manifest indicates the proper minimum OS version, the client will handle the check and enforcement.  And no, it is not in the index at this time.

Reply:
This implementation for now supports all VB+ devices. There is some ongoing work on PWA installs on VB- as well, which can be integrated in the future.